charts/ndm/templates/_helpers.tpl

```diff
@@ -0,0 +1,134 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "ndm.name" -}}
```

Comment:
`ndm` should match the name given in the chart.yaml. `openebs-ndm`.

Reply:
Similar comment applies to the remaining defines as well below in this file.

Reply:
@kmova -- I did not get it. 
Did you mean -- it should be {{define openebs-ndm.name}} ?

Reply:
Yes